channel.py

```diff
@@ -377,6 +377,53 @@ def _payload(self) -> JsonType:
         return {"data": (b64encode(json_minify(payload).encode("utf8"))).decode("utf8")}
 
     async def send_watch(self) -> bool:
+        #-------------------------------------------------
+        # FIX 2024/5
+        #  |
+        #  v
+        try:
+            response: JsonType = await self._twitch.gql_request(        # Gets signature and value
+                GQL_OPERATIONS["PlaybackAccessToken"].with_variables({"login": self._login})
+                )
+        except MinerException as exc:
+            raise MinerException(f"Channel: {self._login}") from exc
+        signature: JsonType | None = response["data"]['streamPlaybackAccessToken']["signature"]
+        value: JsonType | None = response["data"]['streamPlaybackAccessToken']["value"]
+        if not signature or not value:
+            return None
+
+        RequestBroadcastQualitiesURL = f"https://usher.ttvnw.net/api/channel/hls/{self._login}.m3u8?sig={signature}&token={value}"
+
+        try:
+            async with self._twitch.request(                            # Gets list of streams
+                "GET", RequestBroadcastQualitiesURL
+            ) as response1:
+                BroadcastQualities = await response1.text()
+        except RequestException:
+            return False
+        
+        LowestQualityBroadcastURL = BroadcastQualities.split("\n")[-1]  # Just takes the last line, this should probably be handled better in the future
+
+        try:
+            async with self._twitch.request(                            # Gets actual streams
+                "GET", LowestQualityBroadcastURL
+            ) as response2:
+                StreamURLList = await response2.text()
+        except RequestException:
+            return False
+
+        StreamLowestQualityURL = StreamURLList.split("\n")[-2] # For whatever reason this includes a blank line at the end, this should probably be handled better in the future
+
+        try:
+            async with self._twitch.request(                            # Downloads the stream
+                "GET", StreamLowestQualityURL
+            ) as response3:                                             # I lied, well idk, but this code doesn't listen for the actual video data
+                return response3.status == 200
+        except RequestException:
+            return False
+
+        # FIX 2024/5 END
+        #-------------------------------------------------
         """
         This uses the encoded payload on spade url to simulate watching the stream.
         Optimally, send every 60 seconds to advance drops.
```

Comment:
This is a method documentation string. Per PEP 8 style guide, it's supposed to be placed right before the body of the method, not in the middle of it. Please move the code around so that it conforms to the style guide.

Reply:
I'll try.
I'll be honest, I never did anything in python, and I don't plan on doing so again. I hate the lack of closing parenthesis, making me search where a function ends. And whitespace being interpreted, tho good for beginners who need structure, prevents some nicer and more readable formatting in more complex code.

Reply:
Changed it to
`# Start of fix for 2024/5 API Change`
and
`# End of fix for 2024/5 API Change`

Do you want to remove the original code below it too? It can never be reached, but I left it there for your future reference.

Reply:
Changed it again to match other method documentation strings in the files. I found differing information in the PEP 8 style guide, but this should be good now:
```
"""
Start of fix for 2024/5 API Change
"""

Code

"""
End of fix for 2024/5 API Change.
Old code below.
"""
```

Reply:
@DevilXD marking this as resolved, as it now fits other similar comments.